save-core/src/commonNonJsTest/kotlin/org/cqfn/save/core/ValidationTest.kt

```diff
@@ -42,7 +42,7 @@ class ValidationTest {
             assertEquals(
                 """
                     Error: Couldn't find `execCmd` in [general] section of `${generalConfig.configLocation}` config.
-                    Current configuration: execCmd=null, tags=null, description=null, suiteName=null, excludedTests=null, expectedWarningsPattern=null, runConfigPattern=null, timeOutMillis=null
+                    Current configuration: execCmd=null, tags=null, description=null, suiteName=null, excludedTests=null, expectedWarningsPattern=null, expectedWarningsMiddlePattern=null, expectedWarningsEndPattern=null, runConfigPattern=null, timeOutMillis=null
```

Comment:
Can we finally put `$generalConfig.toString()` or something similar here, instead of fixing it manually every time?

Reply:
Yes. Fixed

Reply:
Agree with Peter